modules/cluster-logging-manual-rollout-rolling.adoc

```diff
@@ -183,22 +159,22 @@ $ oc exec elasticsearch-cdm-5ceex6ts-1-dcd6c4c7c-jpw6 -c elasticsearch -- es_uti
   "active_shards_percent_as_number" : 100.0
 }
 ----
-<1> Make sure this parameter is `green` before proceeding.
+<1> Make sure this parameter is `yellow` before proceeding.
 
 . If you changed the Elasticsearch configuration map, repeat these steps for each Elasticsearch pod.
 
 . Once all the deployments for the cluster have been rolled out, re-enable shard balancing:
 +
 [source,terminal]
 ----
-$ oc exec <any_es_pod_in_the_cluster> -c elasticsearch -- es_util --query=_cluster/settings -XPUT 'https://localhost:9200/_cluster/settings' -d '{ "transient": { "cluster.routing.allocation.enable" : "none" } }'
+$ oc exec <any_es_pod_in_the_cluster> -c elasticsearch -- es_util --query=_cluster/settings -XPUT 'https://localhost:9200/_cluster/settings' -d '{ "persistent": { "cluster.routing.allocation.enable" : "all" } }'
```

Comment:
remove the `'http://localhost:9200/_cluster/settings'` and add quotations around the query value... `--query="_cluster/settings"`